scico/linop/radon_svmbir.py

```diff
@@ -30,39 +30,59 @@
     raise ImportError("Could not import svmbir; please install it.")
 
 
-class ParallelBeamProjector(LinearOperator):
-    r"""Parallel beam Radon transform based on svmbir.
+class TomographicProjector(LinearOperator):
+    r"""Tomographic projector based on svmbir.
 
     Perform tomographic projection of an image at specified angles, using
     the `svmbir <https://github.com/cabouman/svmbir>`_ package. The
     ``is_masked`` option selects whether a valid region for projections
     (pixels outside this region are ignored when performing the
     projection) is active. This region of validity is also respected by
     :meth:`.SVMBIRSquaredL2Loss.prox` when :class:`.SVMBIRSquaredL2Loss`
-    is initialized with a :class:`ParallelBeamProjector` with this option
+    is initialized with a :class:`TomographicProjector` with this option
     enabled.
+    Details about the underlying scanner geometries can be found
+    `here <https://svmbir.readthedocs.io/en/latest/overview.html>`.
     """
 
     def __init__(
         self,
         input_shape: Shape,
         angles: np.ndarray,
         num_channels: int,
-        center_offset: Optional[float] = 0.0,
-        is_masked: Optional[bool] = False,
+        center_offset: float = 0.0,
+        is_masked: bool = False,
+        geometry: str = "parallel",
+        dist_source_detector: Optional[float] = None,
+        magnification: Optional[float] = None,
     ):
         """
         Args:
-            input_shape: Shape of the input array.
+            input_shape: Shape of the input array. Can be of length
+                either 2 (2D array) or 3 (3D array). When specifying
+                a 2D array, the format for the input_shape is
+                (num_rows,num_cols). For a 3D array, the format for
+                the input_shape is (num_slices,num_rows,num_cols).
+                Note that (num_rows,num_cols) and (1,num_rows,num_cols)
```

Comment:
Both, I would say.

Reply:
Added some explanations. After a careful look, I could not find places in the svmbir docs that explain the slices, rows, and cols properly

Reply:
It would be really helpful to the user to have some understanding of how the reconstruction shape relates to the sinogram shape. Perhaps ask the svmbir developers or do a simple test to figure out what the relationship is?

Reply:
Made the changes. Added the description of the output shape at the end of the docstring of the `__init__` function. Is there a better place to put it?